metadata-service/restli-client/src/main/java/com/linkedin/entity/client/JavaEntityClient.java

```diff
@@ -285,4 +304,69 @@ public SearchResult filter(@Nonnull String entity, @Nonnull Filter filter, @Null
         int start, int count, @Nonnull String actor) throws RemoteInvocationException {
         return _entitySearchService.filter(entity, filter, sortCriterion, start, count);
     }
+
+    @SneakyThrows
+    @Override
+    public VersionedAspect getAspect(@Nonnull String urn, @Nonnull String aspect, @Nonnull Long version,
+        @Nonnull String actor) throws RemoteInvocationException {
+        return _entityService.getVersionedAspect(Urn.createFromString(urn), aspect, version);
+    }
+
+    @SneakyThrows
+    @Override
+    public VersionedAspect getAspectOrNull(@Nonnull String urn, @Nonnull String aspect, @Nonnull Long version,
+        @Nonnull String actor) throws RemoteInvocationException {
+        return _entityService.getVersionedAspect(Urn.createFromString(urn), aspect, version);
```

Comment:
Is this correct? 

Shouldn't getAspectOrNull try to get a versioned aspect, catch 404, and return null?